.eslintrc

```diff
@@ -4,7 +4,8 @@
   "rules": {
     "import/no-unresolved": 0,
     "flowtype/define-flow-type": 1,
-    "flowtype/use-flow-type": 1
+    "flowtype/use-flow-type": 1,
+	"semi": ["error", "never"]
```

Comment:
Should we use spaces instead of tabs?

Reply:
I think this can be reverted anyway because prettier takes care of it. I actually write the code with semicolons and prettier removes them for me

Reply:
But yes, definetly spaces

Reply:
Using the indent rule would avoid such issue, I have automatic linting fixes on save in VSCode. Plus we would get errors thrown by the hook, should I add it ?

Reply:
We also run prettier in the precommit git hook so it's unnecessary